### ISSUE TITLE
Add info about OSHW certification

### ISSUE THREAD

**Issue by dl1com** [AllYarnsAreBeautiful/allyarnsarebeautiful.github.io#11]
Refer to https://certification.oshwa.org/us002598.html

(@VIPQualityPost is this valid for all hardware flavors? (Shield, Interface, ESP32))

**Comment by VIPQualityPost** [comment]
No, just the ESP32 one.
Do you think we should file for the others as well?

**Comment by t0mpr1c3** [comment]
Only if Victoria going to continue selling the shields, but I would guess there isn't going to be a market for them any more.

**Comment by VIPQualityPost** [comment]
I think she plans to continue selling them for the "DIY" crowd as some people seem to like buying them. I did tell her that I would help migrate the design to KiCad to help with updating the silkscreen etc to make things more clear. It is probably worth re-filing for the shield in particular.

**Comment by VIPQualityPost** [comment]
Once @vthunder finishes the project import, then I'll file the shield design for OSHW cert as well.

**Comment by t0mpr1c3** [comment]
This is ready to go now isn't it?

**Comment by VIPQualityPost** [comment]
Yep, let me add it to the shield design and then we can close this.

**Comment by VIPQualityPost** [comment]
Oh, didn't realize this was on the website repo rather than HW. 
Here's the link: https://certification.oshwa.org/us002734.html
Maybe you can add it where you would like on the website?

**Comment by t0mpr1c3** [comment]
OK
